### ISSUE TITLE
Install siteimprove for QA

### ISSUE THREAD

**Issue by reillyzlab** [CityOfBoston/boston.gov-d8#584]
Must have a homepage and anonymous view set before we can do this.

**Comment by davidrkupton** [comment]
Site is now properly loaded on site improve.
There is a separate issue for integration with drupal which is #364 - adding a siteimprove module.
- this issue can close.
